assets/src/edit-story/components/panels/design/captions/captions.js

```diff
@@ -125,27 +150,53 @@ function CaptionsPanel({ selectedElements, pushUpdate }) {
         !isMixedValue &&
         tracks.map(({ id, trackName }) => (
           <Row key={`row-filename-${id}`}>
-            <BoxedTextInput
-              value={trackName}
-              disabled
-              aria-label={__('Filename', 'web-stories')}
-              clear
-              onChange={() => handleRemoveTrack(id)}
-              key={`filename-${id}`}
-            />
+            <InputRow>
+              <Input
+                value={trackName}
+                aria-label={__('Filename', 'web-stories')}
+                onChange={() => handleRemoveTrack(id)}
+                disabled
```

Comment:
It looks like the colour of the text in the input should be overridden for the `disabled` input here -- it's using the `primary` colour:
<img width="537" alt="Screenshot 2021-03-12 at 09 12 56" src="https://user-images.githubusercontent.com/3294597/110905566-f2b82c80-830a-11eb-85b3-1e0682ebcd84.png">